components/lib/crash-sentry/src/main/java/mozilla/components/lib/crash/sentry/SentryService.kt

```diff
@@ -121,11 +120,8 @@ class SentryService(
     @VisibleForTesting
     internal fun initSentry() {
         SentryAndroid.init(applicationContext) { options ->
-            options.dsn = Uri.parse(dsn).buildUpon()
-                .appendQueryParameter("uncaught.handler.enabled", "false")
-                .build()
-                .toString()
-
+            options.enableUncaughtExceptionHandler = false
```

Comment:
As interactions with the Sentry sdk are static is hard to create a meaningful unit test